build/package/install_agent_spawn_host.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -ex
+
+while getopts 'i:h:t:u:v:' opt; do
+    case $opt in
+        i) keyfile="$OPTARG";;      # SSH identity file
+        u) user="$OPTARG";;         # Username on the remote host
+        h) hostsfile="$OPTARG";;    # Output of Evergreen host.list
+        t) groupid="$OPTARG";;      # APIGroupId
+        v) apiKey="$OPTARG";;       # APIKey
+        *) exit 1;;
+    esac
+done
+
+hosts=$(cat << EOF | python - "$hostsfile"
```

Comment:
[non blocking ]I'd rather not use python but we can look at this later on

Reply:
Could you explain why? I don't see any downside in using python.

Reply:
a couple of reasons
1) we are introducing a dependency on python for a go project, I prefer to stay in language for tools
2) since this going to run mostly on evergreen and we aren't setting anything to have a python available this may break if evergreen decides to change python, which takes me back to point one of introducing a new dependency we need to maintain

This is working now so don't touch it but we may change it later to either be all bash or add some `go` to do it